packages/cursorless-org-docs/src/docs/user/README.md

```diff
@@ -350,7 +350,7 @@ Saying `"every paint"` would select `foo.bar` and `baz|bongo`.
 
 ##### `"short paint"`
 
-The `"short paint"` scope is like `"paint"`, but stops not only at whitespace but also stops if it would exit the nearest [surrounding pair](#surrounding-pair). For example, given the text
+The `"short paint"` scope is like `"paint"`, but stops not only at whitespace but also stops if it would hit the nearest [surrounding pair](#surrounding-pair). For example, given the text
```

Comment:
```suggestion
The `"short paint"` scope is like `"paint"`, but in addition to stopping at whitespace, it will also stop before the nearest [surrounding pair](#surrounding-pair). For example, given the text
```